### DISCUSSION TITLE
Just installed from vcpkg, missing many includes

### DISCUSSION THREAD

**Discussion by robertflesch** [odygrd/quill#587] [General]
So I just installed into visual studio 22, using vcpkg in Windows 11

Instructions on the front page of project say to include
#include "quill/Backend.h"
#include "quill/Frontend.h"
#include "quill/LogMacros.h"
#include "quill/Logger.h"
#include "quill/sinks/ConsoleSink.h"
#include <string_view>

but when I look at what was installed in the include dir, I see is
clock              
detail             
filters            
handlers           
Clock.h            
Config.h           
Fmt.h              
list.txt           
Logger.h           
LogLevel.h         
MacroMetadata.h    
PatternFormatter.h 
Quill.h            
QuillError.h       
TransitEvent.h     
TweakMe.h          
Utility.h          

no sign of backend.h, frontend.h, or other files and dirs.
did I install wrong version?

**Comment by odygrd** [comment]
Hey, yes this looks like very old version like v3 or something. Please install `7.1.0` as this is the latest vcpkg port available, you need to git checkout/pull master for the vcpkg repo
https://github.com/microsoft/vcpkg/blob/master/ports/quill/vcpkg.json#L3

**Comment by robertflesch** [comment]
Thx, that was indeed the problem. 
Just wondering why the default pkg that it is grabbing is 4 versions old!
